### PR TITLE
docs: glossary zkApp account, zkApp Manager account

### PR DESCRIPTION
**Pull request by barriebyron**
This PR adds two new terms to the glossary

Esp interested in @garwalsh's feedback on the capitalization and "productization" of the term: zkApp Manager account
Choices:
- zkApp Manager account
- zkApp manager account
Or? 

See https://github.com/o1-labs/docs2/pull/656/files you can see in the doc preview https://docs2-git-tuto11-minadocs.vercel.app/zkapps/tutorials/advanced-account-updates that we have several variations of spelling, which makes me think we'll need clarity on what this account is and how we consistently refer to it

### CODE REVIEW

File: docs/glossary.mdx [hunk 1]
```diff
@@ -530,6 +530,14 @@ Zero knowledge apps (zkApps) are Mina Protocol's smart contracts powered by zero
 
 A command line tool that zkApp developers use to scaffold and deploy smart contracts. Install the [zkApp CLI](https://www.npmjs.com/package/zkapp-cli).
 
+### zkApp account
+
+A smart contract account. Each zkApp account provides 8 fields of 32 bytes each of arbitrary storage. When a Mina address contains a verification key, it acts as a zkApp account.
+
+### zkApp Manager account
```
Comment (barriebyron) [inline docs/glossary.mdx:537]:
```suggestion
### zkApp manager account
```

**Review by garwalsh — CHANGES_REQUESTED** [review]
zkApp Manager is capitalised in the description, otherwise lgtm!

File: docs/glossary.mdx [hunk 2]
```diff
@@ -530,6 +530,14 @@ Zero knowledge apps (zkApps) are Mina Protocol's smart contracts powered by zero
 
 A command line tool that zkApp developers use to scaffold and deploy smart contracts. Install the [zkApp CLI](https://www.npmjs.com/package/zkapp-cli).
 
+### zkApp account
+
+A smart contract account. Each zkApp account provides 8 fields of 32 bytes each of arbitrary storage. When a Mina address contains a verification key, it acts as a zkApp account.
+
+### zkApp manager account
+
+A specific type of smart contract that manages a particular thing. For example, the zkApp Manager account for a token controls all properties of token accounts and determines rules for token minting, burning, and transfer.
```
Comment (barriebyron) [inline docs/glossary.mdx:539]:
```suggestion
A specific type of smart contract that manages a particular thing. For example, the zkApp manager account for a token controls all properties of token accounts and determines rules for token minting, burning, and transfer.
```

**Comment by barriebyron** [comment]
Thanks @garwalsh  back to you.

**Review by garwalsh — APPROVED** [review]

**Pull request merged**